src/core/or/circuitpadding.h

```diff
@@ -0,0 +1,685 @@
+/*
+ * Copyright (c) 2017, The Tor Project, Inc. */
+/* See LICENSE for licensing information */
+
+/**
+ * \file circuitpadding.h
+ * \brief Header file for circuitpadding.c.
```

Comment:
This could use a comment that explains WTF-PAD 101 for beginners, unless such a comment will appear somewhere else.  Most of the stuff below in this file only makes sense if you know how the rest of wtfpad works.

Reply:
Done in cdbd5c0af8.